website/src/logged_in/navigation/NavBar.js

```diff
@@ -129,6 +132,8 @@ function NavBar(props) {
         <Fragment>
             <AppBar position="sticky" className={props.classes.appBar}>
                 <Toolbar className={props.classes.appBarToolbar}>
+                <h2 style={{ color: 'black' }}>Menu</h2>
```

Comment:
Clicking on "Menu" should pop open the nav bar, like the icon. Please join the two.